Doc/library/locale.rst

```diff
@@ -592,22 +626,23 @@ library.
 Access to message catalogs
 --------------------------
 
-.. function:: gettext(msg)
-.. function:: dgettext(domain, msg)
-.. function:: dcgettext(domain, msg, category)
-.. function:: textdomain(domain)
-.. function:: bindtextdomain(domain, dir)
+.. function:: gettext(msg, /)
+.. function:: dgettext(domain, msg, /)
+.. function:: dcgettext(domain, msg, category, /)
+.. function:: textdomain(domain, /)
+.. function:: bindtextdomain(domain, dir, /)
+.. function:: bind_textdomain_codeset(domain, codeset, /)
```

Comment:
I'm not sure if these `/` args do much to clarify here, shall we revert this bit? I don't think it fixed any warnings?

Reply:
Actual functions in `locale` do not accept named-keywords, only pos-only ones. I think that the docs should reflect that. Why do you think that `/` is not helpful here?

Reply:
We have many function signatures in docs that do not reflect the exact technical reality of the code, but instead simplify them, for readability, brevity, and simplicity.